### PR TITLE
Update subsciption cost and calculation

### PR DESCRIPTION
**Pull request by omrivardi**

### CODE REVIEW

File: lib/src/models/ScyllaCloud.ts [hunk 1]
```diff
@@ -38,6 +38,10 @@ const LicenseCorePriceYearly = {
     reserved: 495 as YearlyPrice
 }
 
+const SubsciptionCostPriceYearly = {
+    onDemand: 837.7 as YearlyPrice,
+    reserved: 560 as YearlyPrice
```
[the remark this answers is not in the document]
Comment (omrivardi) [reply lib/src/models/ScyllaCloud.ts:43]: From our pricing spreadsheet. I will send you a more detailed message

[the remark this answers is not in the document]
Comment (tzach) [reply lib/src/models/ScyllaCloud.ts:43]: What I do not understand where is the old price in the code
I was expecting the old price to be removed

[the remark this answers is not in the document]
Comment (tzach) [reply lib/src/models/ScyllaCloud.ts:43]: Is this new price includes variable costs (e.g. network)

[the remark this answers is not in the document]
Comment (tzach) [reply lib/src/models/ScyllaCloud.ts:43]: 560 * 1.5 = 840, why 837.3?

**Comment by tzach** [comment]
The code seems correct, but something does not adds up.
A minimal cluster of 3 x i3en.xlarge(4 vCPUs, 2.5TB storage)
For yearly (upfront) each node its (4 + 2.5) * 560 * 3 = 10,920 yearly, or 910 monthly.
This is not what this PR return, but I'm not sure why.

**Comment by omrivardi** [comment]
This only covers the subscription price of the instance, you need to add the instance price.
I will double check the pricing. It will be helpful if we have some resource with calculated prices according to the new model and the breakdown

File: lib/src/models/ScyllaCloud.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (omrivardi) [reply lib/src/models/ScyllaCloud.ts:43]: Where 1.5 is coming from? I took it from the spreadsheet

[the remark this answers is not in the document]
Comment (tzach) [reply lib/src/models/ScyllaCloud.ts:43]: Cell B8 On-demand, hourly is 150% of the base all up front

**Comment by tzach** [comment]
Thanks. Indeed, i did not include the VM
With it I get: 

$1,548 (see below) while the calc result it 1,539

Not sure where the 11$ gaps is from.

<google-sheets-html-origin><style type="text/css"></style>

vCPU | 4 | $2,240
-- | -- | --
Storage | 2.5 | $1,400
EC2 |   | $2,550
Total |   | $6,190
Monthly |   | $516
Nodes | 3 | $1,548

**Comment by tzach** [comment]
@omrivardi did you include the volume discount in the calc and UI?
AFAIU, you should *not*

**Comment by tzach** [comment]
We need to add a statement "Pricing from Nov 15, 2021"

**Comment by tzach** [comment]
LGTM

**Pull request merged**